src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -55,7 +56,8 @@ export class TSConfigReader extends OptionsComponent
             return;
         }
 
-        var data = require(fileName);
+        var result = ts.readConfigFile(fileName, (fileName) => FS.readFileSync(fileName, 'utf8'));
```

Comment:
Should we use an arrow function so that `readFile` has the proper `this` binding?  `sys` could theoretically be an instance of a class that expects proper `this` values.